src/lib.rs

```diff
@@ -448,4 +453,16 @@ mod tests {
             Err(ConfigFetchIssue::NoConfigFound)
         ));
     }
+
+    #[test]
+    fn name_from_url() {
+        assert_eq!(
+            repo_name_from_url("https://github.com/hyper-type/hahmlet/"),
+            Some("hahmlet"),
+        );
+        assert_eq!(
+            repo_name_from_url("https://github.com/hyper-type/Advent"),
+            Some("Advent"),
+        );
```

Comment:
I think this means we get a collision if font A has org1/a_name as repo and font B has org2/a_name. While perhaps unlikely I tend to think this would be a valid config.

Reply:
Agree this is possible, but unlikely. Opened #4 to track.